src/main/java/com/iggroup/universityworkshopmw/domain/model/Client.java

```diff
@@ -15,6 +15,6 @@
    private String clientId;
    private String userName;
    @Setter
-   private double profitAndLoss;
+   private Double funds;
```

Comment:
I made funds here double and not Double, as Double takes up more space (java is efficient etc I know). I thought Double, Integer etc are used primarily in DTOs, and use int, double elsewhere, unless there is some reason for it

Reply:
Okay sure. I can change that